config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/custom-parameters.json

```diff
@@ -3,6 +3,64 @@
     "description": "The namespace where you want to create the deployment.",
     "displayname": "Namespace",
     "name": "namespace",
+    "templateKeyName": [
```

Comment:
`"templateKeyName"` field is not required for config and secret parameters. Please remove it for all parameters in `custom-parameters.json` file.

Reply:
done